freqtrade/plugins/pairlist/AgeFilter.py

```diff
@@ -86,14 +98,17 @@ def _validate_pair_loc(self, pair: str, daily_candles: Optional[DataFrame]) -> b
             return True
 
         if daily_candles is not None:
-            if len(daily_candles) >= self._min_days_listed:
+            if len(daily_candles) >= self._min_days_listed and \
+                    len(daily_candles) <= self._max_days_listed:
                 # We have fetched at least the minimum required number of daily candles
                 # Add to cache, store the time we last checked this symbol
                 self._symbolsChecked[pair] = int(arrow.utcnow().float_timestamp) * 1000
                 return True
             else:
                 self.log_once(f"Removed {pair} from whitelist, because age "
                               f"{len(daily_candles)} is less than {self._min_days_listed} "
-                              f"{plural(self._min_days_listed, 'day')}", logger.info)
+                              f"{plural(self._min_days_listed, 'day')} or more than "
+                              f"{self._max_days_listed} {plural(self._max_days_listed, 'day')}",
+                              logger.info)
```

Comment:
Same as above, basically - `because age 3 is less than 10 days or more than None days` does not look that nice ...